docs/organizations/using-coding-standards.md

```diff
@@ -1,41 +1,43 @@
-
-Optionally, set a default coding standard to apply it automatically to all new repositories that you add to Codacy. If no coding standard is set as default, new repositories don't follow any coding standard and use the Codacy default configurations instead.
-
-!!! important
-    Coding standards turn tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
+-   [Creating a coding standard](#creating)
+-   [Editing a coding standard](#editing)
+-   [Deleting a coding standard](#deleting)
+-   [Defining a default coding standard](#set-default)
+-   [Using a coding standard alongside tool configuration files](#using-with-tool-configuration)
 
 ## Creating a coding standard {: id="creating"}
 
+!!! info "Codacy currently supports up to 10 coding standards per organization"
+
 To create a coding standard for your organization:
 
 1.  Open your organization **Policies** page, tab **Coding standards**.
 
-1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens a window with the coding standard creation form.
-
-    !!! note
-        Codacy currently supports up to 10 coding standards per organization.
+1.  Click the button **Create new standard** at the top right-hand corner of the page. This opens the coding standard creation form.
 
 1.  Enter a unique name and click **Create standard**.
 
-    Optionally, select a repository for Codacy to use as a baseline when bootstrapping the tool and pattern configurations for the new coding standard. This is useful if you already have a configured repository that you wish to use as a template.
+    Optionally, select a repository to use as a baseline for the new coding standard.
 
     ![Creating a new coding standard](images/coding-standard-create.png)
 
 1.  Select the programming languages that the new coding standard should cover and click **Next: Tools and patterns**.
 
-    The coding standard will only include configurations for the tools that support at least one of the selected languages.
+    !!! important
+        The coding standard will only control tools that can analyze one or more of the languages selected in this step.
+         
+        If a language is not selected in this step, the associated tools are not necessarily disabled. Rather, Codacy uses default tool configurations, potentially enabling the tools and leading to unintended analysis of the omitted language. To override this behavior:
 
-    ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
+        1.  When creating a coding standard, select all the languages used in the relevant repositories, so that all associated tools are controlled by the coding standard.
```

Comment:
why not suggest selecting all the languages?

I'm wondering if there is a use case for allowing users to select the languages? (other than showing a shorter list)

Reply:
I don't have a strong opinion either way. Selecting all is a bit "noisier" but definitely more thorough.

Using "all" for now, but it would make sense to consider showing the default tool status in the UI, and potentially update the filtering mechanism to always show all tools and filter by language with a dropdown, instead of a list of checkboxes on a previous step